doc/running_on_phone.md

```diff
@@ -35,21 +42,63 @@ Files required to boot from eMMC are:
 
 ### Uploading it to the eMMC
 
--  Use the `pureflash` command to burn image into the eMMC card `sudo pureflash <PurePhone.img> /dev/sdX` **Note:** Replace `/dev/sdX` with the disk name you found for your phone after connecting it to your computer.
+####  via bootloader
+
+On linux you can use:
+1. `pureflash`
+
+Fastest command to burn image into the eMMC card `sudo pureflash <PurePhone.img> /dev/sdX` **Note:** Replace `/dev/sdX` with the disk name you found for your phone after connecting it to your computer.
+
+2. `dd` or `etcher`
+
+You can use any raw memory copy tool such as `dd` or `etcher` which will transfer whole image built to the device memory. It will take much longer because it will copy whole 16GB memory.
```

Comment:
```suggestion
You can use any raw memory copy tool such as `dd` or `etcher` which will transfer the whole image built to the device memory. It will take much longer because it will copy the whole 16GB of memory.
```